polytope.yml

```diff
@@ -33,6 +33,11 @@ modules:
         - { name: OPENAI_API_KEY, value: "#pt-secret openai-api-key" }
         - { name: ANTHROPIC_API_KEY, value: "#pt-secret anthropic-api-key" }
         - { name: FOOTWAY_API_KEY, value: "#pt-secret footway-api-key" }
+        - { name: SINCH_API_KEY_ID, value: "#pt-secret sinch-api-key-id" }
+        - { name: SINCH_API_KEY_SECRET, value: "#pt-secret sinch-api-key-secret" }
+        - { name: SINCH_API_PROJECT_ID, value: "#pt-secret sinch-api-project-id" }
+        - { name: SINCH_APP_ID, value: "#pt-secret sinch-app-id" }
+        - { name: SINCH_SMS_SENDER, value: "#pt-secret sinch-sms-sender" }
```

Comment:
I'm a bit reluctant to accept this at this point because this step will refuse to run if these secrets haven't been stored. Could you set these to empty strings instead? That way people who want to work with Sinch can still do so without it breaking things for others